src/java/org/apache/cassandra/replication/MutationTrackingService.java

```diff
@@ -325,6 +363,57 @@ public boolean registerMutationCallback(ShortMutationId mutationId, IncomingMuta
         return incomingMutations.subscribe(mutationId, callback);
     }
 
+    public void executeTransfers(String keyspace, Set<SSTableReader> sstables, ConsistencyLevel cl)
+    {
+        shardLock.readLock().lock();
+        try
+        {
+            logger.info("Creating tracked bulk transfers for keyspace {} sstables {}", keyspace, sstables);
+
+            KeyspaceShards shards = checkNotNull(keyspaceShards.get(keyspace));
+            CoordinatedTransfers transfers = CoordinatedTransfers.create(keyspace, shards, sstables, cl);
+            logger.info("Split input SSTables into transfers {}", transfers);
```

Comment:
I see the transfer ID is in the `toString()` for `CoordinatedTransfer`. I suppose this will be the piece of information that will later allow us to trace everything important about the transfer across it's lifecycle across all the replicas on our log exploration system of choice?